pkg/find/vulnerabilities/trivy/scanner.go

```diff
@@ -27,12 +27,8 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-type Config interface {
```

Comment:
This interface is supposed to defined other configuration options for vulnerability scanner, which are not currently implemented. The imageRef is only one property that might be configured.

What's more, it allows us to implement the interface segregation principle (part of SOLID). When we inject very generic starboard.Config interface into each scanner, we're violating this rule. In other words, we don't want to inject configuration of Polaris into a vulnerability scanner.